uniprotkb-rest/src/main/java/org/uniprot/api/uniprotkb/groupby/service/GroupByECService.java

```diff
@@ -11,6 +8,9 @@
 import org.uniprot.api.uniprotkb.service.UniProtEntryService;
 import org.uniprot.core.cv.ec.ECEntry;
 
+import java.util.*;
```

Comment:
Can you please run maven command to make sure that import order is based on maven plugin?